### PR TITLE
New modes@ at#1 revise facility values, add parking visibility

### PR DESCRIPTION
**Pull request by nick-knowles**
Revise parking add attributes to PArking Prioperty so as to allow   Allow virtual stations (ParkingVisibility) . Aligh PARKING vehicleTypes enum with VEHICLE parking typoes enum.  Revise CarSreviceFacilitie sto include valet service.

### CODE REVIEW

**Review by Aurige — APPROVED** [review]

File: xsd/netex_framework/netex_reusableComponents/netex_vehicleType_support.xsd [hunk 1]
```diff
@@ -479,6 +482,7 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="moped"/>
 			<xsd:enumeration value="motorcycle"/>
 			<xsd:enumeration value="quadbike"/>
+						<xsd:enumeration value="car"/>
```
Comment (skinkie) [inline xsd/netex_framework/netex_reusableComponents/netex_vehicleType_support.xsd:485]: Indentation?

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_parking_support.xsd [hunk 2]
```diff
@@ -381,8 +392,8 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="motorwayParking"/>
 			<xsd:enumeration value="roadside"/>
 			<xsd:enumeration value="parkingZone"/>
-			<xsd:enumeration value="undefined"/>
 			<xsd:enumeration value="cycleRental"/>
+			<xsd:enumeration value="undefined"/>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_parking_support.xsd:396]: Change not required.

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_parking_support.xsd [hunk 3]
```diff
@@ -436,17 +447,21 @@ Rail transport, Roads and Road transport
 	</xsd:simpleType>
 	<xsd:simpleType name="ParkingVehicleEnumeration">
 		<xsd:annotation>
-			<xsd:documentation>Allowed values for PARKING VEHICLE types.</xsd:documentation>
+			<xsd:documentation>Allowed values for PARKING VEHICLE types. </xsd:documentation>
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_parking_support.xsd:450]: Space suffix

File: xsd/netex_part_1/part1_ifopt/netex_ifopt_parking_version.xsd [hunk 4]
```diff
@@ -620,6 +623,16 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Maximum allowed Stay as Duration.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="BayGeometry" type="BayGeometryEnumeration" minOccurs="0">
```
Comment (skinkie) [inline xsd/netex_part_1/part1_ifopt/netex_ifopt_parking_version.xsd:626]: What does the +v1.2.2 mean?

I also notice two spaces opposed to the single space in +v1.1 below

**Comment by Aurige** [comment]
Agreed to be merged during meeting on April 21st in order to move forward

**Pull request merged**